lib/kuzzle.js

```diff
@@ -377,21 +378,34 @@ async function dumpAndExit(kuzzle, suffix) {
 function initVault (vaultKey, secretsFile) {
   const
     defaultEncryptedSecretsFile =
-      path.resolve(`${__dirname}/../../config/secrets.enc.json`),
+      path.resolve(`${__dirname}/../config/secrets.enc.json`),
     encryptedSecretsFile =
       secretsFile || process.env.KUZZLE_SECRETS_FILE || defaultEncryptedSecretsFile;
 
   let key = vaultKey;
-  if ((!_.isString(vaultKey) || vaultKey.length <= 0)
-     && _.isString(process.env.KUZZLE_VAULT_KEY)
-     && process.env.KUZZLE_VAULT_KEY.length > 0
-  ) {
+  if (_.isEmpty(vaultKey) && !_.isEmpty(process.env.KUZZLE_VAULT_KEY)) {
     key = process.env.KUZZLE_VAULT_KEY;
   }
+  const
+    fileExists = fs.existsSync(encryptedSecretsFile),
+    vault = new Vault(key, null, encryptedSecretsFile);
+
+  if (!_.isEmpty(secretsFile) || !_.isEmpty(process.env.KUZZLE_SECRETS_FILE)) {
+    console.log({ fileExists, encryptedSecretsFile})
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a semicolon at the end of this statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ASemicolon)